app/src/main/java/com/yml/womensafety/navigationdrawer/home/OnBoardingFragment1.kt

```diff
@@ -0,0 +1,18 @@
+package com.yml.womensafety.navigationdrawer.home
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import com.yml.womensafety.R
+import com.yml.womensafety.authentication.LoginActivity
+import kotlinx.android.synthetic.main.fragment_on_boarding1.*
+
+class OnBoardingFragment1 : Fragment(R.layout.fragment_on_boarding1) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        btnSkip1.setOnClickListener {
```

Comment:
Rename btn name as btnSkipOne instead. Same applies in other places

Reply:
Done